blueocean-dashboard/src/main/js/components/records.jsx

```diff
@@ -23,7 +23,7 @@ export class PipelineRecord extends Record({
     weatherScore: 0,
 }) {
     isFolder() {
-        return this.numberOfFolders !== null;
+        return this._class === 'io.jenkins.blueocean.service.embedded.rest.PipelineFolderImpl';
```

Comment:
@vivek at the moment this is explicitly for filtering out what we know is vanilla folders, vs positively identifying things based on capabilities, so I think it will be ok for this as it won't stop new types showing up, just identify them by folders. 

I believe we have some tickets open for capabilities, if not, we should open them for sure.